app/views/landing-pages/parents/PageParents.vue

```diff
@@ -1179,6 +1178,16 @@ export default {
   margin-bottom: 20px;
 }
 
+.concept-covered-tab.beginner img{
+  max-width: 92px;
+}
+
+@media screen and (min-width: 700px) {
+  .concept-covered-tab.beginner .topics, .concept-covered-tab.intermediate .topics{
```

Comment:
You covered `.beginner` and `.intermediate` selector. Do we also need the third `.advanced` selector here?

Reply:
No we don't need .advanced as of now because we are adding gap between them.